apps/web/components/Organization/NewsArticle/NewsArticle.tsx

```diff
@@ -39,25 +39,25 @@ export const NewsArticle: React.FC<
       </Box>
       <Box className="rs_read">
         <Text variant="intro" as="p" paddingBottom={2}>
-          {newsItem.intro}
+          {newsItem?.intro}
         </Text>
       </Box>
-      {Boolean(newsItem.image) && (
+      {Boolean(newsItem?.image) && (
         <Box
           paddingY={2}
           className={cn({
-            [styles.floatedImage]: newsItem.fullWidthImageInContent === false,
+            [styles.floatedImage]: newsItem?.fullWidthImageInContent === false,
           })}
         >
           <Image
-            {...newsItem.image}
-            url={newsItem.image.url + '?w=774&fm=webp&q=80'}
-            thumbnail={newsItem.image.url + '?w=50&fm=webp&q=80'}
+            {...newsItem?.image}
+            url={newsItem?.image?.url + '?w=774&fm=webp&q=80'}
+            thumbnail={newsItem?.image?.url + '?w=50&fm=webp&q=80'}
           />
         </Box>
       )}
       <Box className="rs_read" paddingBottom={4} width="full">
-        {webRichText(newsItem.content as SliceType[], {
+        {webRichText(newsItem?.content as SliceType[], {
```

Comment:
Perhaps we'd want to fallback to an empty array like we've been doing in other places in this PR :)

Reply:
Yeah agree, good point